docs/tutorials/runtime-benchmarking/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Introduction
+---
+
+# Benchmark `pallet-template`
```

Comment:
??

Reply:
Keeping this file as placeholder for now so no dead link and pass the CI test.